.github/workflows/ci.yml

```diff
@@ -65,10 +65,10 @@ jobs:
           cache: 'pip'
       - name: Install package
         run: |
-          pip install -e '.[docs]'
+          - run: pip install -r docs/requirements.txt -e .
```

Comment:
Maybe undo these changes? This makes the YAML file invalid.

Reply:
Thanks!